### PR TITLE
Feature std dev

### PR DESCRIPTION
**Pull request by Anna-MarieSeelen**
did some things

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (LottePronk) [inline inflammation/models.py:49]: Try to find a clearer variable name instead of 'mmm'.

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (LottePronk) [inline inflammation/models.py]:
```suggestion
    s_dev = sum(devs) / len(data)
```

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    return {'standard deviation': s_dev2}
```
Comment (LottePronk) [inline inflammation/models.py]:
```suggestion
    return {'standard deviation': s_dev}
```

File: tests/test_models.py [hunk 4]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
+def test_daily_standard_deviation(data, expected_standard_deviation):
```
Comment (LottePronk) [inline tests/test_models.py:40]: Try to add a docstring that explains what cases are being tested.

File: inflammation-analysis.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (LottePronk) [inline inflammation-analysis.py]:
```suggestion
        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), 's_dev': models.s_dev(inflammation_data)}
```

File: inflammation/models.py [hunk 6]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
```
Comment (LottePronk) [inline inflammation/models.py:50]: Rename 'devs' to a clearer variable name.

File: inflammation/models.py [hunk 7]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (LottePronk) [inline inflammation/models.py:47]: Try to use a more conventional abbreviation for standard deviation (std for example).

File: tests/test_models.py
Comment (LottePronk) [inline tests/test_models.py:1]: Test the following: 
negative values
edge cases 
multiple patients

**Review by LottePronk — CHANGES_REQUESTED** [review]
Good luck!

**Review by LottePronk — APPROVED** [review]
Thanks for doing all this hard work.
